Formula/ssh-agent-filter.rb

```diff
@@ -19,7 +19,22 @@ def install
   end
 
   test do
-    system "#{bin}/ssh-agent-filter", "-V"
+    (testpath/"test.sh").write <<-EOS.undent
+      mydir=$(pwd)
+      cd $(mktemp -d)
+      ssh-keygen -q -N '' -f keyA
```

Comment:
@tiwe-de sorry, I'm not sure I understand what that is. Could you give me a pointer?